projects/components/src/datagrid/datagrid.component.html

```diff
@@ -1,4 +1,42 @@
 <clr-datagrid [clrDgLoading]="isLoading" [ngClass]="this.clrDatagridCssClass" (clrDgRefresh)="gridStateChanged($event)">
+    <clr-dg-action-bar *ngIf="buttonConfig">
+        <div class="btn-group" *ngFor="let button of buttonConfig.globalButtons">
+            <button
+                class="btn"
+                [ngClass]="button.class"
+                *ngIf="isButtonShown(button)"
+                (click)="button.handler()"
+                [disabled]="isButtonDisabled(button, button.isActive())"
+            >
```

Comment:
This should not have spaces around the text, then you can fix any `trim()`s in tests
```typescript
>{{
 button.label 
}}</button>
```